src/page_components/listing/ListingCard.tsx

```diff
@@ -115,6 +116,14 @@ const ListingCard = (props: ListingCardProps) => {
     }
   }
 
+  const getTableSubHeader = (subheader: ListingCardHeader, returnElement?: boolean) => {
+    return React.isValidElement(subheader.content) && returnElement ? (
```

Comment:
@emilyjablonski @ColinBuyck Thoughts on implementing this without the `returnElement` prop? Would there be any reason we would want to wrap an element with a paragraph tag?